Build.proj

```diff
@@ -254,7 +253,7 @@
 
   <Target Name="_PackageInspectorNuGetWin" DependsOnTargets="Xamarin_Build_ReadAllProperties">
     <PropertyGroup>
-      <InspectorClientFilesPath>Clients\Xamarin.Interactive.Client.Windows\bin\$(Configuration)$(InspectorOutputPathSuffix)\</InspectorClientFilesPath>
+      <InspectorClientFilesPath>Clients\Xamarin.Interactive.Client.Windows\bin\$(Configuration)\Inspector\</InspectorClientFilesPath>
```

Comment:
Note that since you changed the output path, you need to fix the installer entries for `Xamarin Inspector.exe*` (just two files).

Reply:
Yes. WIP.